core/build.gradle.kts

```
@@ -187,6 +194,65 @@ ide {
     generatedSource(javaCCTest, "test")
 }
 
+fun configureAnnotationSet(task: JavaCompile, sourceSet: SourceSet) {
+    with(task) {
+        source = sourceSet.java
+        classpath = sourceSet.compileClasspath
+        destinationDirectory.set(options.generatedSourceOutputDirectory)
+        options.compilerArgs.add("-proc:only")
+        org.gradle.api.plugins.internal.JvmPluginsHelper.configureAnnotationProcessorPath(sourceSet, sourceSet.java, options, project)
+    }
+}
```

```suggestion
fun JavaCompile.configureAnnotationSet(sourceSet: SourceSet) {
    source = sourceSet.java
    classpath = sourceSet.compileClasspath
    destinationDirectory.set(options.generatedSourceOutputDirectory)
    options.compilerArgs.add("-proc:only")
    org.gradle.api.plugins.internal.JvmPluginsHelper.configureAnnotationProcessorPath(sourceSet, sourceSet.java, options, project)
}
```

`destinationDirectory` should be "directory with the generated class files", so `options.generatedSourceOutputDirectory` looks wrong here.